### DISCUSSION TITLE
Raspberry Pi

### DISCUSSION THREAD

**Discussion by klox51** [janhq/jan#4235] [Get Help]
Hello,
is-it possible ton install jan on a Raspberry pi with the Raspberry Pi on it ?
If it's possible on which version of the raspberry Pi is-ti available (3, 4, 5) ?
Thank you

**Comment by niggiGCH** [comment]
hi communtity
I'm wondering the same thing! Any suggestions? thanks in advance

**Comment by eckartal** [comment]
Hey @klox51, @niggiGCH.

[Jan](https://jan.ai/) and [Cortex](https://cortex.so/) aren't compatible with Raspberry Pi due to its ARM64 architecture, while Cortex is built for x86_64 (AMD64).

It's in on our roadmap - we'll have good news soon!
